src/bin/tikv-ctl.rs

```diff
@@ -106,7 +106,7 @@ fn new_debug_executor(
             ))) as Box<DebugExecutor>
         }
         (Some(remote), None) => Box::new(new_debug_client(remote, mgr)) as Box<DebugExecutor>,
-        _ => unreachable!(),
+        _ => unreachable!("You have to set up the parameter --pb or --db!"),
```

Comment:
Maybe "Missing required parameters: --pb or --db" is a better message? It is not common for error messages to speak to you like their friend. :)